app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= stylesheet_link_tag 'header' %>
+<header class="header">
+  <% if user_signed_in? %>
+    <div class="header-user-container">
+      <div>
+        <%= current_user.name %>
+      </div>
+      <div>
+        <%= current_user.surname %>
+      </div>
+      <%= link_to "Sign out", "/user/sign_out", :method => :delete %>
+    </div>
+  <% else %>
+    <div>Login</div>
```

Comment:
https://github.com/DmitriyFirsov/twitter-app/pull/4/files#diff-6819a296bd0ea8218c483ace46886affd40c53d158e8e675c96de9aa7a99df44R20

You also have a "Log in" invitation, so maybe you move it to a partial and use the same markup everywhere?